scripts/dmc_reset.py

```diff
@@ -275,7 +279,7 @@ def main():
                 logger.debug(f"Found {dut['platform']} with JTAG ID {dut['id']}")
                 break
 
-    ret = reset_dmc(args)
+    ret = reset_dmc(vars(args))
```

Comment:
Would be kind of nice to just keep args here, I think. There is an alternative though (which I'll mention where it's needed).